scripts/pre-commit.sh

```diff
@@ -26,4 +26,6 @@ GO_SOURCE_FILES=$(find elasticdl/pkg -name '*.go' -print0 | tr '\0' ' ')
 
 YAML_SOURCE_FILES=$(find .travis.yml -name '*.yml' -print0 | tr '\0' ' ')
 
-pre-commit run --files "$PYTHON_SOURCE_FILES $GO_SOURCE_FILES"
+MARKDOWN_SOURCE_FILES=$(find elasticdl -name '*.md' -print0 | tr '\0' ' ')
```

Comment:
This skips over the /docs directory. Not right.

If you are afraid of the messy output from currently non-compliant Markdown files, the trick you need is

https://github.com/sql-machine-learning/sqlflow/blob/f566a9362f52d1ce66d3354526f4e118db1f3564/scripts/pre-commit/shellcheck.sh#L15

Reply:
In order to run pre-commit only on the changed files in this pull request, update the pre-commit.sh as the following, and it works.

```BASH
PROJECT_URL=https://github.com/sql-machine-learning/elasticdl.git
REMOTE=$(git remote -v | grep $PROJECT_URL | cut -f1 | head -1)
pre-commit run --from-ref "$REMOTE/develop" --to-ref HEAD
```

Reply:
After we update the format of all the markdown files, we can update the pre-commit command to check all the files in this repo.